lib/ukallocbbuddy/Config.uk

```diff
@@ -1,6 +1,15 @@
-config LIBUKALLOCBBUDDY
+menuconfig LIBUKALLOCBBUDDY
 	bool "ukallocbbuddy: Binary buddy page allocator"
 	default n
 	select LIBNOLIBC if !HAVE_LIBC
 	select LIBUKDEBUG
 	select LIBUKALLOC
+
+if LIBUKALLOCBBUDDY
+	config LIBUKALLOCBBUDDY_SMP_SAFE
+	bool "SMP safe allocator"
```

Comment:
Hi @razvanvirtan, You do not need to define a separate flag to enable SMP safe bbuddy allocator. This is because there exists a global flag `CONFIG_HAVE_SMP` .
The spinlock library internally uses this flag. If the CONFIG_HAVE_SMP is not set then it will map the functions to the dummy spinlock functions. Hence you do not need to handle SMP case